src/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.product__photo {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.product__img {
+  width: 100%;
+  height: 100%;
+}
+
+.product__name {
+  margin: 0;
+  margin-bottom: 5px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.product__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.product__stars {
+  display: flex;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background: url("./images/star-active.svg") no-repeat center;
+}
+
+.stars__item:last-child {
```

Comment:
add classes from stars block task, so if I change the class of your element in dev tools I will see how the count of active stars changing.